File: Frontend/xml-angular-app/src/app/model/opste/uspesna-transformacija.ts

export interface UspesnaTransformacija {
    odgovor: string
}

export function napraviUspesnuTransformaciju(zahtevJson): UspesnaTransformacija {
  console.log(zahtevJson)
  return {
    odgovor: zahtevJson.$.odgovor
  }
}

File: Backend/xml.project/src/main/java/com/example/xml/project/response/UspesnaTransformacija.java
package com.example.xml.project.response;

import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class UspesnaTransformacija {

    @XmlAttribute
    private byte[] odgovor;

    public UspesnaTransformacija() {

    }

    public UspesnaTransformacija(final byte[] odgovor) {
        this.odgovor = odgovor;
    }

    public byte[] isOdgovor() {
        return odgovor;
    }
}
